src/tools/csg_imc_solve.cc

```diff
@@ -104,4 +107,19 @@ void CG_IMC_solve::Run() {
   x.y() = -inverse * A.transpose() * B.y();
 
   x.Save(outputfile);
```

Comment:
@DRosen285 is the output of the complete solution vector still useful if we also write the potentials?

Reply:
If with potentials you mean the potential update (dU or dpot) as well, then I think this can go away.

Reply:
Then we should move this, too!

Reply:
Done.